src/components/api-type-test.ts

```diff
@@ -0,0 +1,63 @@
+import * as express from "express";
+import { components } from "../types/api";
+
+const list_results: components["schemas"]["Result"][] = [];
+
+const app = express();
+app.use(express.json());
+
+// get all results
+app.get("/v1alpha2/parents/:parent/results", (req, res) => {
+    const parent = req.params.parent
+
+    console.log("parent", parent);
+
+    console.log(list_results);
+  res.json(list_results);
+});
+
+  app.post("/v1alpha2/parents/:parent/results/:result_uid", (req, res) => {
+
+    const result_uid = req.params.result_uid
```

Comment:
I understand its only for testing the types, but can you fix that intention anyway? :)